src/collection-facets/more-facets-content.ts

```diff
@@ -129,28 +131,30 @@ export class MoreFacetsContent extends LitElement {
     }
   }
 
+  /**
+   * Whether facet requests are for the search_results page type (either defaulted or explicitly).
+   */
+  private get isSearchResultsPage(): boolean {
+    return [undefined, 'search_results'].includes(
```

Comment:
q: why not === ?

s: pageType, can we make that a type something for all to share? and also so we know what's allowed-ish

Reply:
Ah yeah, `search_results` is the default page type so we need to check against `undefined` here too. There's a `PageType` type we can use from the search service here, so I'll update the typing of this array. Thanks!

Reply:
Updated for clarity & type correctness